apps/judicial-system/web/src/components/CaseDates/CaseDates.tsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react'
 import { useIntl } from 'react-intl'
 
-import { CaseDecision, CaseType } from '@island.is/judicial-system/types'
+import { CaseDecision } from '@island.is/judicial-system/types'
 import { formatDate } from '@island.is/judicial-system/formatters'
 import { Box, Button, IconMapIcon, Text } from '@island.is/island-ui/core'
 import { TIME_FORMAT } from '@island.is/judicial-system/consts'
 import { caseDates } from '@island.is/judicial-system-web/messages'
+import { CaseType } from '@island.is/judicial-system-web/src/graphql/schema'
```

Comment:
Just wondering, was there a specific reason for switching to a generated CaseType in relation to the traffic violation screen task ? Or could they have been separate PRs?

Reply:
They could very well be separate PRs. My thinking is that we do this refactoring as we are working on other stories, that way we don't need to plan and estimate them, but rather we just refactor as we go along. We can discuss other strategies for the future.